pandas-dev/pandas-stubs
# 0704issues

- [x] Closes multiple issues
    - Close #80 - DataFrame constructor should accept np.ndarray for index
    - Close #81 - quantile should accept np.ndarray
    -  Close #84 - reindex needed arguments
    -  Close #85 - Series.iloc[] should accept np.ndarray[int]
    -  Close #86 - Series.iloc[np.ndarray[int]] should return Series
    - `test_frame.py`
        - Modify `test_types_quantile()` to check for `np.ndarray()`
        - add `test_frame_index_numpy()`
        - add `test_frame_reindex()`
    - `test_series.py`
        - add `test_iloc_ndarray()`
